packages/framework/src/Model/Pricing/Currency/Currency.php

```diff
@@ -19,6 +19,7 @@ class Currency
     public const ROUNDING_TYPE_INTEGER = 'integer';
 
     public const DEFAULT_EXCHANGE_RATE = '1';
+    public const CZK_EXCHANGE_RATE_TO_EUR = '0.04';
```

Comment:
Personally, I would rather see this constant in the DataFixture class. 
This may imply it's ok to put fixed exchange rates into application or use this one in the application.

Reply:
good point, fixed in a3e9ddb51